src/penn_chime/presentation.py

```diff
@@ -252,12 +240,7 @@ def display_n_days_slider(st, p: Parameters, d: Constants):
 
 def show_more_info_about_this_tool(
     st,
-    recovery_days,
-    doubling_time,
-    r_naught,
-    relative_contact_rate,
-    doubling_time_t,
-    r_t,
+    parameters,
```

Comment:
Maybe just p instead of parameters

Reply:
this is actually a preference of mine. part of it is that i worked with a linter for a long time that disabled one letter variables. the other part of it is that i like calling on keywords `parameters=p` better than `p=p`. (I know it's not uniform across the PR, which is my mistake). but above all, the i think entries like `doubling_time=parameters.doubling_time` are easier to read than `doubling_time=p.doubling_time`.

i'm not _married_ to it. do you think there's a big add from keeping it uniform w the one-letter variable that's already all over the codebase?

Reply:
You are right about the linter. Why couldn't I thought of a shorter name for the thing...

This is just fine.

I was probably thinking about interpolating f strings directly. That would get quite long with the long variable name.